src/main/java/com/hotels/housekeeping/service/impl/FileSystemHousekeepingService.java

```diff
@@ -15,11 +15,9 @@
  */
 package com.hotels.housekeeping.service.impl;
 
-import static java.lang.String.format;
-
-import java.io.IOException;
-import java.util.List;
-
+import com.hotels.housekeeping.model.LegacyReplicaPath;
```

Comment:
The original formatting looks ok, this doesn't. You may need to adjust your settings.

Reply:
+1